demisto_sdk/commands/init/initiator.py

```diff
@@ -327,6 +391,59 @@ def get_object_id(self, created_object: str):
                         input(f"Please enter the id name for the {created_object}: ")
                     )
 
+    def get_pack_path(self) -> str:
+        """Gets the pack path from current path."""
+        if (
+            self.xsiam
+            and self.output
+            and re.search(INTEGRATIONS_DIR_REGEX, self.output)
+        ):
+            if result := re.search(self.PACK_PATH_REGEX, self.output):
+                return result[0]
+        return self.output
+
+    def create_initiators_and_init_modeling_parsing_rules(
+        self, product: Optional[str], vendor: Optional[str]
+    ) -> bool:
+        """Creates initiators object for modeling and parsing rules init and initialize them.
+
+        Args:
+            product (str): The product from the user.
+            vendor (str): The vendor from the user.
+        """
+        output_path = self.get_pack_path()
+        self.full_output_path = output_path
+        modeling_rules_initiator = Initiator(
+            output=os.path.join(self.full_output_path, MODELING_RULES_DIR),
+            common_server=self.common_server,
+            dir_name=str(self.dir_name),
+            template=self.HELLO_WORLD_MODELING_RULES,
+            script=False,
+            integration=False,
+            demisto_mock=self.demisto_mock,
+            xsiam=self.xsiam,
+            name=self.dir_name,
+        )
+        parsing_rules_initiator = Initiator(
```

Comment:
To avoid all this duplication in the initiator, why not do something like this?
```def create_initiator(output_path, template, is_pack=False):
    return Initiator(
        output=os.path.join(self.full_output_path, output_path),
        common_server=self.common_server,
        dir_name=str(self.dir_name),
        template=template,
        is_pack=is_pack,
        script=False,
        integration=False,
        demisto_mock=self.demisto_mock,
        xsiam=self.xsiam,
        name=self.dir_name,
    )
```
And then call the initiator and create whatever you need:
```
modeling_rules_initiator = create_initiator(MODELING_RULES_DIR, self.HELLO_WORLD_MODELING_RULES)
parsing_rules_initiator = create_initiator(PARSING_RULES_DIR, self.HELLO_WORLD_PARSING_RULES, is_pack=False)
```